src/cpu/callback.cpp

```diff
@@ -154,29 +155,54 @@ const char* CALLBACK_GetDescription(callback_number_t cb_num)
 	return CallBack_Description[cb_num].c_str();
 }
 
-callback_number_t CALLBACK_SetupExtra(callback_number_t cb_num, Bitu type, PhysPt physAddress, bool use_cb = true)
+static uint8_t callback_setup_extra(const callback_number_t callback_number,
+                                    const uint8_t callback_type,
+                                    const PhysPt start_address,
+                                    const bool should_use_callback = true)
 {
-	if (cb_num >= CB_MAX)
+	if (callback_number >= CB_MAX) {
+		E_Exit("Unknown callback number 0x%04x", callback_number);
```

Comment:
Would it be better to just log an error and return without hard exiting? I really dislike hard exits.

Reply:
OK, changed - no more `E_Exit` in the reworked function.